src/renderer/chrome-api.js

```diff
@@ -64,7 +69,35 @@ exports.injectTo = function (extensionId, isBackgroundPage, context) {
       chrome.runtime.incrementOriginResultID()
     },
 
-    query() { },
+    insertCSS(tabId, details, cb) {
```

Comment:
Why only stub this function? Just for the record.

Reply:
Grammarly checks availability of API methods and uses callbacks for chaining.